### ISSUE TITLE
About Masked Contrastive Pre-Training

### ISSUE THREAD

**Issue by Maelic** [YangLing0818/SGDiff#8]
Dear authors,
Thanks for sharing this amazing work, are you planning to also share the training code for the Masked Contrastive Pre-Training part? This would be very helpful to correctly reproduce the full approach.
Thank you

**Comment by YangLing0818** [comment]
> Dear authors, Thanks for sharing this amazing work, are you planning to also share the training code for the Masked Contrastive Pre-Training part? This would be very helpful to correctly reproduce the full approach. Thank you

Thanks for your interests in our work, we may release the training code for Masked Contrastive Pre-Training after a period of time.

**Comment by YangLing0818** [comment]
> Dear authors, Thanks for sharing this amazing work, are you planning to also share the training code for the Masked Contrastive Pre-Training part? This would be very helpful to correctly reproduce the full approach. Thank you

We have released the code of Masked Contrastive Pre-Training part, please check out.

**Comment by Maelic** [comment]
Thanks, I will check it out.
